### PR TITLE
Minor fixes and polish for plotting

### PR DESCRIPTION
**Pull request by imsodin**
I recently for the first time used pyvista for plotting and discovered a few minor things to fix/improve (looking at the PRs using pyvista mainly as a nicer vtk interface seems to be a minority use :) ).

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -1639,9 +1639,7 @@ def add_volume(self, volume, scalars=None, clim=None, resolution=None,
             Scalars used to "color" the mesh.  Accepts a string name of an
             array that is present on the mesh or an array equal
             to the number of cells or the number of points in the
-            mesh.  Array should be sized as a single vector. If both
-            ``color`` and ``scalars`` are ``None``, then the active scalars are
```
Comment (akaszynski) [inline pyvista/plotting/plotting.py]: I think leaving this in is important as the user needs to know that active scalars are used if no scalars are entered in, but we should remove `color` as it's not a valid kwarg.

Also, could you fix "UnformGrid" on line 1635? Looks like that's a misspelling.

File: pyvista/plotting/plotting.py [hunk 2]
```diff
@@ -1846,7 +1837,7 @@ def add_volume(self, volume, scalars=None, clim=None, resolution=None,
             return actors
 
         if not isinstance(volume, pyvista.UniformGrid):
-            raise TypeError('Type ({}) not supported for volume rendering at this time. Use `pyvista.UniformGrid`.')
+            raise TypeError('Type {} not supported for volume rendering at this time. Use `pyvista.UniformGrid`.'.format(type(volume)))
```
Comment (akaszynski) [inline pyvista/plotting/plotting.py]: Good catch.  Perhaps change to:
```python
raise TypeError('Type "{}" not supported for volume rendering at this time. Use `pyvista.UniformGrid`.'.format(type(volume)))
```

**Comment by akaszynski** [comment]
Also, I noticed you're force pushing.  Be careful when you do that:
https://softwareengineering.stackexchange.com/questions/312691/is-it-wrong-to-git-push-force-branches

File: pyvista/plotting/plotting.py (hunk 2 above)
Comment (imsodin) [reply pyvista/plotting/plotting.py]: The output of type looks like `<class '...'>`, so additional stuff around it seems superfluous to me.

Comment (akaszynski) [reply pyvista/plotting/plotting.py]: Good point.

**Pull request merged**